pylint/test/unittest_checkers_utils.py

```diff
@@ -6,88 +6,84 @@
 
 """Tests for the pylint.checkers.utils module."""
 
-import unittest
-import warnings
-
 import astroid
 
 from pylint.checkers import utils
-from pylint import __pkginfo__
-
-class UtilsTC(unittest.TestCase):
+import pytest
 
-    def test_is_builtin(self):
-        self.assertEqual(utils.is_builtin('min'), True)
-        self.assertEqual(utils.is_builtin('__builtins__'), True)
-        self.assertEqual(utils.is_builtin('__path__'), False)
-        self.assertEqual(utils.is_builtin('__file__'), False)
-        self.assertEqual(utils.is_builtin('whatever'), False)
-        self.assertEqual(utils.is_builtin('mybuiltin'), False)
+def test_is_builtin():
+    assert utils.is_builtin('min') is True
+    assert utils.is_builtin('__builtins__') is True
+    assert utils.is_builtin('__path__') is False
+    assert utils.is_builtin('__file__') is False
+    assert utils.is_builtin('whatever') is False
+    assert utils.is_builtin('mybuiltin') is False
```

Comment:
Not sure if we need the `is True` / `is False` here (though I also wonder why the old code didn't use `assertTrue`/`assertFalse`)